src/taipy/_run.py

```diff
@@ -65,6 +65,8 @@ def _run(*apps: t.List[_AppType], **kwargs) -> t.Optional[Flask]:
         return app.run(**kwargs)
 
 
+# Avoid black adding too many empty lines
+# fmt: off
```

Comment:
This is to avoid conflict between black versions.

On python 3.10, black would only add 1 empty line after line 75.
But on Github action, black would add 2, which causes the conflict.